test/test_suite.py

```diff
@@ -123,3 +123,15 @@ def _test_get_table_oid(self, table_name, schema=None):
         insp = inspect(meta.bind)
         oid = insp.get_table_oid(table_name, schema)
         self.assert_(isinstance(oid, int))
+
+class HANAConnectUrlUserHDBUserStoreTest(fixtures.TestBase):
+
+    @testing.only_on('hana')
+    @testing.only_if('hana+hdbcli')
```

Comment:
What happens with PyHDB, could you create a test for that and raise a exception that hdbuserstore keys are not supported with PyHDB?

Reply:
code change and now included in the PR

Reply:
Thank you.